man/journald.conf.xml

```diff
@@ -381,6 +383,16 @@
         <literal>=</literal> and the following argument, true is assumed. Otherwise, the argument is parsed
         as a boolean.</para>
 
+        <para>The socket forwarding address can be specified with the credential
+        <literal>journald.forward_to_socket</literal>. The following socket types are supported:</para>
+
+        <para><simplelist type="inline">
+          <member><constant>AF_INET</constant> (e.g. <literal>192.168.0.11:4444</literal>)</member>
+          <member><constant>AF_INET6</constant> (e.g. <literal>[2001:db8::ff00:42:8329]:4444</literal>)</member>
+          <member><constant>AF_UNIX</constant> (e.g. <literal>/run/host/journal/socket</literal>)</member>
+          <member><constant>AF_VSOCK</constant> (e.g. <literal>vsock:2:1234</literal>)</member>
+        </simplelist></para>
```

Comment:
I have the suspicion that people will be irritated that you can enable the concept via a config option, but not the target address?